types/play-sound/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for play-sound 1.1
+// Project: https://github.com/shime/play-sound
+// Definitions by: Vincent <https://github.com/Vinnl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.3
+
+/// <reference types="node" />
+
+import { ChildProcess } from "child_process";
+
+interface Player {
+  play(path: string, onError?: (error: null | (Error & { killed: boolean })) => void): ChildProcess;
+  play(path: string, args: Record<string, any>, onError?: (error: null | (Error & { killed: boolean })) => void): ChildProcess;
+}
+
+declare function getPlayer(): Player;
```

Comment:
it seems to accepts options:
https://github.com/shime/play-sound/blob/master/index.js#L50
the options are fully optional, but should be there:
https://github.com/shime/play-sound/blob/master/index.js#L16
https://github.com/shime/play-sound#examples